cypress/support/config/services.js

```diff
@@ -13,6 +14,12 @@ export default {
         asset:
           Cypress.env('APP_ENV') === 'live' ? 'c5ll353v7y9o' : 'c6v11qzyv8po',
       },
+      nonExistentarticle:
+        Cypress.env('APP_ENV') === 'live'
+          ? undefined
```

Comment:
You could add this non-existent asset for all the services rather than using `undefined`. 
```suggestion
          ? { asset: 'c0000000000o' }
```

Reply:
The reason i added them as undefined instead of providing an non existent value is for services like igbo,persian,pidgin and yoruba non-existent error page for now renders the error message in english instead of localised error messages which would make the all of the tests fail except should return a 404 error code.

Reply:
Currently for Igbo, Pidgin and Yoruba, with the locally-running application, I see the localised error messages. So I'm not sure what you're referring to? 
<img width="1368" alt="Screen Shot of Yoruba error page" src="https://user-images.githubusercontent.com/3028997/61636977-a1b77880-ac8e-11e9-85dd-bbce611665cc.png">

Reply:
You see them locally for  Igbo,Pidgin and yoruba and not on test and live env so i have provided npon existent asset value for local env and undefined for test and Live.Same for Persian too but for news i could provide non existent asset value.

Reply:
Yes please add one for News.
So `undefined` means the error page is not on the environment yet, but an asset value means that the error page *is* on that environment. Adding a comment in this file to explain that would add clarity for those reading and maintaining the tests.

Reply:
Sure.

Reply:
I have added for news but haven't modified for the three services Igbo,Pidgin and yoruba as the config file doesnt seem to have the `lang` key and also the error page doesn't have the language attribute.May be could add them when everything is in place.

Reply:
I have updated the config file with the `lang` key for these services and the error tests are enabled for these services too.